gui/api/resources.py

```diff
@@ -920,12 +922,24 @@ def _get_children(self, bundle, vol, children, uid):
                     data['used_pct'],
                 )
                 data['avail'] = humanize_size(data['avail'])
-                
+
                 data['readonly'] = self.__zfsopts.get(
                     child.path,
                     {},
                 ).get('readonly', ('', '-'))[1]
 
+                data['comments'] = self.__zfsopts.get(
+                    child.path,
+                    {},
+                ).get('org.freenas:description', ('', '-'))[1]
+
+                # Filtering out comments field for Jails
+                for jail in jails_list:
+                    child_name = child.name.split('/')
+                    length = len(child_name)
+                    if jail.jail_host == child_name[length - 1] or "warden" in child_name[length - 1]:
+                        data['comments'] = '-'
+
```

Comment:
No, this doesn't make sense.

You should be looking at the source attribute of the property looking for inherited ones and discard it for org.freenas:description

.e.g.
# zfs get -o source 'org.freenas:description zmedia/jails
SOURCE
inherited from zmedia

If source starts with "inherited" for that property, do not show it.